DB.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO `martciv_maria`.`user` (`name`, `age`, `sex`, `ordered_tickets`) VALUES ('Ann', '20', 'women', '13');
+INSERT INTO `martciv_maria`.`user` (`name`, `age`, `sex`, `ordered_tickets`) VALUES ('Bob', '25', 'men', '7');
+INSERT INTO `martciv_maria`.`user` (`name`, `age`, `sex`, `ordered_tickets`) VALUES ('Mary', '36', 'women', '5');
+INSERT INTO `martciv_maria`.`user` (`name`, `age`, `ordered_tickets`) VALUES ('Ket', '18', '20');
```

Comment:
INSERT INTO `martciv_maria`.`user` (`name`, `age`, `sex`, `ordered_tickets`) 
VALUES 
('Ann', '20', 'women', '13'),
('Bob', '25', 'men', '7'),
('Mary', '36', 'women', '5'),
('Ket', '18', '20');

Reply:
The field 'sex' can be null, in case if someone doesn't want to indicate gender. just like in the last case ('Ket', '18', '20');

Reply:
oops :) got it
INSERT INTO martciv_maria.user (name, age, sex, ordered_tickets)
VALUES
('Ann', '20', 'women', '13'),
('Bob', '25', 'men', '7'),
('Mary', '36', 'women', '5'),
INSERT INTO martciv_maria.user(name, age, ordered_tickets) VALUES ('Ket', '18', '20');
It looks simpler.